### PR TITLE
docs: change wrong param name in time_bucket_gapfill finish param description

### PR DESCRIPTION
**Pull request by vonnue-yehya**

### CODE REVIEW

File: api/_hyperfunctions/time_bucket_gapfill/time_bucket_gapfill.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ api_details:
           The end of the period to gapfill.
           Values after `finish` are passed through, but no gapfilling is performed.
           Use `INTEGER` only if your time column is integer-based.
-          We recommend using a `WHERE` clause rather than specifying `start`, which is a legacy option.
+          We recommend using a `WHERE` clause rather than specifying `finish`, which is a legacy option.
```
Comment (billy-the-fish) [inline api/_hyperfunctions/time_bucket_gapfill/time_bucket_gapfill.md]: Best practice is to use the `WHERE` clause. Specifying `finish` is out of date.

**Review by billy-the-fish — CHANGES_REQUESTED** [review]
Sorry to be annoying :-(.

**Comment by billy-the-fish** [comment]
@vonnue-yehya, could you sign the CLA please so we can merge your PR.

**Review by billy-the-fish — APPROVED** [review]

**Comment by vonnue-yehya** [comment]
@billy-the-fish Signed the CLA :)

**Pull request merged**